css-three-column-layout/index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Three Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+
+  <div class="container">
+
+    <div class="row">
+      <div class="column-full">
+        <h1>Designed for life science. Optimized for SaMD and Digital Health.</h1>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="column-one-fourth">
+        <div>
+          <img src="images/merge.png" alt="Merge">
+        </div>
+      </div>
+      <div class="column-one-fourth">
+        <div>
+          <img src="images/cloud-computing.png" alt="Cloud Computing">
+        </div>
+      </div>
+      <div class="column-one-fourth">
+        <div>
+          <img src="images/discussion.png" alt="Discussion">
+        </div>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="column-one-fourth desc-header">
+        <h2>Streamlined QMS</h2>
+        <p>A QMS that inclues modules for all stages of the product life cycle
+          from Design Control to CAPA
+        </p>
+      </div>
+      <div class="column-one-fourth desc-header">
+        <h2>Integrates with existing tools</h2>
+        <p>Enzyme QMS integrates with your existing tools, such as JIRA, GitHub
+          and Jenkins with Enzyme
+        </p>
+      </div>
+      <div class="column-one-fourth desc-header">
+        <h2>In-depth Onboarding</h2>
+        <p>We train your entire team and provide you with support when and as it's
+          needed
+        </p>
+      </div>
+    </div>
+
```

Comment:
Try reorganizing this into just one row of three columns, each with an image, heading, and paragraph in it.